### DISCUSSION TITLE
Is this library compatible with fiber framework ?

### DISCUSSION THREAD

**Discussion by arihantdaga** [go-pkgz/auth#121] [Q&A] [answered]
Is this library compatible with fiber framework ?

**Comment by umputun** [comment, marked as answer]
this library provides nothing custom, but rather stdlib controllers and middleware. Any semi-decent framework should be compatible (in some way) with stdlib.

I'm not familiar with fiber enough but it seems to have its own handlers with some `fiber.Ctx` passed in. In my opinion, it is a terrible idea and I won't touch such a framework, but you do you. Probably they should have a way to adopt the normal handler, but you should check it with fiber's support.
